setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
+name = sphinxcontrib-towncrier
+url = https://github.com/sphinx-contrib/towncrier
```

Let's specify extra URLs that will show up in the sidebar on PyPI:
```suggestion
url = https://github.com/sphinx-contrib/towncrier
project_urls = 
    GitHub: repo = https://github.com/sphinx-contrib/towncrier
    GitHub: issues = https://github.com/sphinx-contrib/towncrier/issues
```